### PR TITLE
New branch

### LINKED ISSUE #105: Add refresh button on jobs page

### PR DESCRIPTION
**Pull request by KianShah**

### CODE REVIEW

**Comment by KianShah** [comment]
pls don't merge, just approve after reading through my changes. This is the current version deployed on heroku

File: server/routes/JobRoutes.js
Comment (robertmzm) [inline server/routes/JobRoutes.js:47]: clean up console

**Review by robertmzm — APPROVED** [review]

**Review by aestrr — APPROVED** [review]

**Pull request merged**
